README.md

```diff
@@ -0,0 +1,57 @@
+# Laser Runner Removal
+
+Laser Runner Removal is a project by the California Strawberry Commission for automated detection and cutting of strawberry runners (also known as stolons).
+
+## Environment setup
+
+1.  Install [ROS 2](https://docs.ros.org/en/foxy/Installation/Ubuntu-Install-Debians.html). Once installed, run:
+
+        $ source /opt/ros/foxy/setup.zsh
+
+2.  Create ROS workspace
+
+        $ mkdir -p ~/ros2_ws/src
+        $ cd ~/ros2_ws
+
+3.  Create Python virtual environment
+
+        $ python3 -m venv venv  --system-site-packages --symlinks
+        $ touch venv/COLCON_IGNORE
+        $ source venv/bin/activate
+
+4.  Add the following to ~/.zshrc
```

Comment:
Do you know anything about the pros and cons of say .sh versus .zshrc files? Again, no strong opinions, wondering more for personal interest.

Reply:
It's definitely personal choice, I've been using zsh forever (mostly because it's the default for mac). It's highly customizable, and I've created a zshrc that I just copy over for any new machine so everyhting looks and works the same 
https://github.com/genkikondo/dotfiles